open-machine-learning-jupyter-book/assignments/prerequisites/python-programming-basics.ipynb

```diff
@@ -1,12 +1,27 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def string_upper_cased(string):
</pre><p>same as above, <code>string_upper()</code></p><p>Please apply this way to name all your new created functions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Did you address this? </p>